### ISSUE TITLE
Windows version of Betty can't find assets\version file

### ISSUE THREAD

**Issue by uwagura** [bartfeenstra/betty#1734]
When trying to run the windows executable version of Betty, you get an error that says ` Failed to execute the script 'main' due to unhandled exception: [Errno 2] No such file or directory: '<path-to-betty>\\betty-windows\\betty\\_internal\\betty\\assets\\VERSION' `

Here's the full traceback:
```
Traceback (most recent call last):
  File "betty\_package\pyinstaller\main.py", line 6, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "betty\gui\app.py", line 29, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "betty\gui\serve.py", line 16, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "betty\extension\__init__.py", line 11, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "betty\extension\nginx\__init__.py", line 10, in <module>
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "betty\cli.py", line 267, in <module>
  File "betty\asyncio.py", line 66, in wait_to_thread
  File "betty\asyncio.py", line 94, in return_value
  File "betty\asyncio.py", line 102, in _run
  File "betty\about.py", line 31, in version_label
  File "betty\about.py", line 18, in version
  File "aiofiles\base.py", line 98, in __aenter__
  File "aiofiles\threadpool\__init__.py", line 94, in _open
  File "concurrent\futures\thread.py", line 58, in run
FileNotFoundError: [Errno 2] No such file or directory: '<path-to-betty-install>\betty-windows\\betty\\_internal\\betty\\assets\\VERSION'

```

**Comment by bartfeenstra** [comment]
@uwagura Thank you for reporting this bug! I opened https://github.com/bartfeenstra/betty/pull/1814 that should fix the problems you have been experiencing.

**Comment by bartfeenstra** [comment]
I tested the fix manually on Windows, and it works. Merged, and released as Betty 0.3.11 soon

**Comment by bartfeenstra** [comment]
Betty 0.3.11 is now available at https://github.com/bartfeenstra/betty/releases/tag/0.3.11
